### ISSUE TITLE
Cookbook incompatibility with yum update

### ISSUE THREAD

**Issue by kkeane** [chef-cookbooks/httpd#26]
On CentOS (and probably other platforms) this cookbook is partially incompatible with yum update.

The problem is that the RPMs will attempt to restart the service "httpd", which is hardcoded. Restarting fails. Yum still continues, but updates will not be automatically applied.

When there are multiple instances of httpd, there obviously is no good solution for this problem.

When there is only a single instance, I would like to have a way to use the standard CentOS default instance, instead of the new httpd-default instance.

**Comment by svanzoest** [comment]
Hi @kkeane 
We strive pretty hard to be compatible with yum updates. Do you have a log file you could share to shows what you are referring to?

**Comment by svanzoest** [comment]
I had a comment here, but deleted it as it referred to a different cookbook. Sorry for the confusion.

**Comment by kkeane** [comment]
I saw your earlier comment and it prompted me to revisit the issue. I had only seen it out of the corner of my eye when I ran "yum update" on one of my servers. Turns out that there was no httpd update yesterday. It must have been some other RPM that depends on httpd and restarts it. Unfortunately, I don't recall which of my servers was the culprit. It could have been owncloud, zarafa server, nagios, or a few other packages that were actually responsible for trying to restart "httpd".
